runtimes/eden/src/pallets_util.rs

```diff
@@ -105,7 +105,7 @@ parameter_types! {
 	pub const ClassDeposit: Balance = 10 * constants::NODL;
 	pub const InstanceDeposit: Balance = 100 * constants::MILLI_NODL;
 	pub const MetadataDepositBase: Balance = 10 * constants::NODL;
-	pub const MetadataDepositPerByte: Balance = 1 * constants::NODL;
+	pub const MetadataDepositPerByte: Balance = constants::NODL;
```

Comment:
I had suggested this change before. The thing is in this case clippy seems to be wrong. There is no extra overhead or unnecessary multiplication after this line is compiled, I have tested it. Then it goes back to the matter of style, which some prefer consistency (so this line looks similar to the previous lines), some would prefer conciseness. I'm not strongly in favour or against any of these two. I like to not see the clippy warning any way.

Reply:
I can silence this warning insted